dali/operators/math/expressions/arithmetic.cc

```diff
@@ -59,7 +59,7 @@ More formally using BNF grammar:
 <scalar>  ::= "$" <uint> ":" <type>
 <uint>    ::= unsigned integer
 <type>    ::= uint8 | uint16 | uint32 | uint64 | int8 | int16 | int32 | int64
-            | float16 | float32 | float64
+| float16 | float32 | float64
```

Comment:
I believe this whole part (starting from `<expr>...`) should be 4-spaces indented with single line brakes around, because right now it doesn't look too appealing (I'm not pushing this change, hardly anyone looks at it :) ):
![image](https://user-images.githubusercontent.com/7101064/101558511-66baff80-39bf-11eb-8b97-7ae864eff955.png)

Reply:
Done